tests/validation/tests/v3_api/resource/terraform/rke2/windows_worker/join-rke2.tftpl

```diff
@@ -0,0 +1,10 @@
+<powershell>
+# Set default shell to powershell
+New-ItemProperty -Path "HKLM:\SOFTWARE\OpenSSH" -Name DefaultShell -Value "C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe" -PropertyType String -Force
+Add-WindowsCapability -Online -Name OpenSSH.Server~~~~0.0.1.0
+Start-Service sshd
+Set-Service -Name sshd -StartupType 'Automatic'
+Write-Output "[INFO] Dowloading setup script..."
+Invoke-WebRequest -Uri https://raw.githubusercontent.com/mdrahman-suse/rke2/md_windows/windows/rke2-standalone-install.ps1 -Outfile C:\Users\Administrator\rke2-standalone-install.ps1
```

Comment:
Let's get this PRed and merged into `rancher/rke2` before merging this PR so that we can pull from there instead of your fork.

Reply:
We still need to do this